tekton-example/README.md

```diff
@@ -56,33 +55,38 @@ kubectl create configmap promote-configmap --from-file=gitconfig
 ```
 This will store your GitHub username and email address in key-value pairs that can be used in the PipelineRun. 
 
-- Edit `service-promote-pipeline-run.yaml` to contain the name of the repository you want to promote to
-
 ## Execute Pipeline
 
-`service-promote-pipeline-run` is designed to build your microservice from its development repository and then promote the new configuration to a GitOps repository (representing a different environment, for example development, staging, test or production).
+The PipelineRun you will create is designed to build your microservice from its development repository and then promote the new configuration to a GitOps repository (representing a different environment, for example development, staging, test or production).
 
-- Create `service-promote-pipeline-run`:
+- To create the PipelineRun, use:
 ```shell
-kubectl create -f service-promote-pipeline-run.yaml
+tkn pipeline start service-promote-pipeline --resource git-source=git-app-repo --resource docker-image=docker-app-image--param commitID=v1 --param github-secret=promote-secret --param github-config=promote-configmap --param to=https://github.com/REPLACE_ME.GITHUB_USERNAME/REPLACE_ME.GITHUB_REPO.git --workspace name=repo-space,clainName=repopvc,subPath=dir -s demo
```

Comment:
`s/clainName/claimName`

Reply:
You can also use `--showlog` on the `tkn pipeline start` command and it will save you having to `tkn pipelinerun logs -f` in the step below.